packages/wallet-adapter-react/src/PrivacyPolicy.tsx

```diff
@@ -0,0 +1,85 @@
+import { Slot } from "@radix-ui/react-slot";
+import { ReactNode, forwardRef } from "react";
+import { SmallAptosLogo } from "./graphics/SmallAptosLogo";
+
+export const PRIVACY_POLICY_URL = "https://aptoslabs.com/privacy";
+
+export interface PrivacyPolicyProps {
+  /** A class name for styling the element. */
+  className?: string;
+  /**
+   * Whether to render as the child element instead of the default element provided.
+   * All props will be merged into the child element.
+   */
+  asChild?: boolean;
+  children?: ReactNode;
+}
+
+const Root = forwardRef<HTMLDivElement, PrivacyPolicyProps>(
+  ({ className, asChild, children }, ref) => {
+    const Component = asChild ? Slot : "div";
+
+    return (
+      <Component ref={ref} className={className}>
+        {children}
+      </Component>
+    );
+  }
+);
+Root.displayName = "PrivacyPolicy.Root";
+
+const Disclaimer = forwardRef<HTMLSpanElement, PrivacyPolicyProps>(
+  ({ className, asChild, children }, ref) => {
+    const Component = asChild ? Slot : "span";
+
+    return (
+      <Component ref={ref} className={className}>
+        {children ?? "By continuing, you agree to Aptos Labs'"}
+      </Component>
+    );
+  }
+);
+Disclaimer.displayName = "PrivacyPolicy.Disclaimer";
+
+const Link = forwardRef<HTMLAnchorElement, PrivacyPolicyProps>(
+  ({ className, asChild, children }, ref) => {
+    const Component = asChild ? Slot : "a";
+
+    return (
+      <Component
+        ref={ref}
+        className={className}
+        href={PRIVACY_POLICY_URL}
```

Comment:
we should accept it as a prop?
```suggestion
        href={privacyPolicyURL ?? PRIVACY_POLICY_URL}
```

Reply:
We could, but I'm not sure of the benefit considering this is specifically `AptosPrivacyPolicy.Link`